Package.swift

```diff
@@ -38,13 +38,33 @@ let package = Package(
                 .product(name: "Vapor", package: "vapor"),
             ]
         ),
-        .testTarget(name: "AppTests", dependencies: [
-            "Gen",
-            .target(name: "App"),
-            .product(name: "XCTVapor", package: "vapor"),
+        .executableTarget(
+            name: "BuildTimeEval",
+            dependencies: [
+                "Gen",
+                .product(name: "Vapor", package: "vapor"),
+                .product(name: "PklSwift", package: "pkl-swift"),
+            ]
```

Comment:
Let's do the same thing as Go, where we bundle the binary data into the executable.

We can do that using `resources`:

```suggestion
            ],
            resources: [
                .copy("../../pkl/dev/config.msgpack")
            ]
```

Reply:
I opted for `embedInCode` here, which is even more similar to pkl-go-examples